src/dialects/abstract/query-generator.d.ts

```diff
@@ -89,6 +91,19 @@ export interface CreateSchemaQueryOptions {
   charset?: string;
 }
 
+export interface CreateTableQueryOptions {
+  collate?: string;
+  charset?: string;
```

Comment:
Nit: could probably extend a few other interfaces here, IIRC collate/charset are somewhere else

Reply:
Yeah, there's quite some duplication in these but might be better to work on those when we migrate the functions to TypeScript